### ISSUE TITLE
register

### ISSUE THREAD

**Issue by Pub4Game** [EvolSoft/ServerAuth#12]
Please make registration only with such characters [a-z].[0-9],[A-Z]

**Comment by Flavius12** [comment]
What do you mean? Do you want to limit the password to these characters?

**Comment by Pub4Game** [comment]
Yes, the players did not created a password - "pass_-s word"

**Comment by Flavius12** [comment]
Hmm...
Interesting idea...
A ServerAuth extension may do this

**Comment by Pub4Game** [comment]
Do this as a separate function that can be enabled or disabled in the configuration

**Comment by Pub4Game** [comment]
@Flavius12 It will be in 2.00?

**Comment by Flavius12** [comment]
No. This feature will be added on a different plugin (a ServerAuth extension)

**Comment by Pub4Game** [comment]
When you make such a plugin?

**Comment by Flavius12** [comment]
When I have time

**Comment by Pub4Game** [comment]
Well, thanks for all the bug fixes (ServerAuth)

**Comment by Flavius12** [comment]
You are welcome :+1:

**Comment by legoboy0215** [comment]
Yay! Bugs are mostly gone!!!

**Comment by Flavius12** [comment]
@legoboy0215 Yes!!! :smiley:. Did you find one?

**Comment by legoboy0215** [comment]
No.... I have school. :P

**Comment by legoboy0215** [comment]
Can you help us with Slapper BTW? (Off-Topic)

**Comment by Flavius12** [comment]
I've school too. I won't develop anymore because of school. I will be back on development the next summer

**Comment by legoboy0215** [comment]
Noooo....

**Comment by Pub4Game** [comment]
Nooo.... :~

**Comment by Pub4Game** [comment]
You'll update the plugin ServerKits?

**Comment by legoboy0215** [comment]
I don't use it :).... Thats is the only one I don't use.

**Comment by Pub4Game** [comment]
I like it ServerKits, but there's no time limit :( Please update the plugin :)

**Comment by Flavius12** [comment]
@Pub4Game I've school. I won't develop anymore because of school. I will be back on development the next summer and maybe also on Christmas holidays

**Comment by legoboy0215** [comment]
Yay! See you! You in... University? :P

**Comment by Flavius12** [comment]
Not, 3rd year of high school :p

**Comment by legoboy0215** [comment]
Oooo.... 1st year of pre-school for me... :P Just Joking.

**Comment by TheDiamondYT1** [comment]
Could you please add the feature to register by typing pass in chat? (And logging in)

**Comment by PEMapModder** [comment]
@Pub4Game Why do you want that to happen? To make it easier to brute force the password??

**Comment by Pub4Game** [comment]
@PEMapModder No, what would the players create the normal passwords that could remember.

**Comment by PEMapModder** [comment]
"Eazy to re ember" is quite easy to remember too, while actually secure.

**Comment by Pub4Game** [comment]
For you yes, but other players can't....

**Comment by Flavius12** [comment]
@Pub4Game Sorry but which player is too stupid to write a difficult password with strange characters? More than the 90% of users use a password with only letters and numbers

**Comment by Flavius12** [comment]
@RandomAltThingy **Please look** at https://github.com/EvolSoft/ChatLogin

**Comment by Pub4Game** [comment]
Believe me in Russia there are such players :D
